lib/utils.ts

```diff
@@ -63,6 +63,11 @@ const ASYNC_UTILS = [
   'waitForDomChange',
 ];
 
+const ASYNC_EVENTS = [
```

Comment:
This are SYNC_EVENTS actually, right?

Reply:
I wouldn't say there are events either - there are helpers or utilities.. Events would be "tab", "input", "change", "click" and so on... 🤔

Reply:
initially thought it should be no-await-sync-utils, but feel not so correct...I can rename it to utils if it makes more sense.